backend/fms_core/migrations/0018_v3_1_0.py

```diff
@@ -270,5 +270,64 @@ class Migration(migrations.Migration):
             field=models.ForeignKey(null=False, blank=True, on_delete=models.PROTECT,
                                     related_name='fms_core_individual_modification', to=settings.AUTH_USER_MODEL),
         ),
-
+        migrations.AlterField(
```

Comment:
These are modifications to 3.1, yet this branch will be merged with master for 3.2 ?